manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```diff
@@ -47,7 +47,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.3.0
+          image: quay.io/k8scsi/csi-provisioner:v1.4.0
```

Comment:
I remember that there are some bugs on csi-provisioner:v1.3.x.
We need 1.4 +, or latest, 1.4.0 maybe ok for us.